src/utils/getUserEmail.ts

```diff
@@ -0,0 +1,19 @@
+import { getUserEmailFromColaJwt } from '@co-digital/login';
+import { log } from '../utils/logger';
+import { Request } from 'express';
+
+import * as config from '../config';
+
+export const getUserEmail = (req: Request) => {
+    if (config.NODE_ENV === 'production') {
+        const jwt = req.signedCookies[config.COOKIE_ID_NAME];
```

Comment:
We should not handle the cookies directly in the app. Let's go with it for now, but we will add a new utility on   `@co-digital/login` module

Reply:
Thanks @DanielMurray97